### PR TITLE
Change request domain character limit to 3 chars

### PR DESCRIPTION
**Pull request by Siegrift**
Changed the limit of 7 characters in request name subpage and instant registration (dev only endpoint).

### CODE REVIEW

**Comment by Siegrift** [comment]
Mhh, this seems not to be working just yet.

**Comment by Siegrift** [comment]
Ok. Works now.

File: src/name_bazaar/ui/pages/instant_register_name_page.cljs [hunk 1]
```diff
@@ -2,13 +2,19 @@
...
     [district0x.ui.components.transaction-button :refer [transaction-button]]
+    [name-bazaar.shared.constants :refer [supported-tld-length?]]
     [name-bazaar.ui.components.app-layout :refer [app-layout]]
     [name-bazaar.ui.components.ens-record.ens-name-input :refer [ens-name-input]]
     [name-bazaar.ui.utils :refer [valid-ens-subname? path-for]]
     [re-frame.core :refer [subscribe dispatch]]
     [reagent.core :as r]
     [soda-ash.core :as ui]))
 
+(defn- valid-tld? [tld-name]
```
Comment (ladislavdubravsky) [inline src/name_bazaar/ui/pages/instant_register_name_page.cljs:13]: :+1:

**Review by ladislavdubravsky — APPROVED** [review]
LGTM!

**Pull request merged**